cmd/exec/exec.go

```diff
@@ -95,6 +95,8 @@ func buildExecutionPlan(onebuildConfig config.OneBuildConfiguration, commands ..
 }
 
 func bashCommand(s *sh.Session, command string) *sh.Session {
+	configFileAbsoluteDir, _ := config.GetAbsoluteDirFromConfigFile()
```

Comment:
function name could be `GetAbsoluteDirPathOfConfigFile`

Reply:
Good idea.